packages/react-ds/src/components/Pagination/Pagination.tsx

```diff
@@ -2,25 +2,25 @@ import React, { useState, useEffect, useCallback } from 'react'
 
 import classNames from 'classnames/bind'
 
-import { IPagination } from '@venice/core/models'
-import styles from '@venice/styles/components/Pagination.module.scss'
-
 import {
   getRangeIndexes,
   hasNextRange,
   hasPreviousRange,
-} from './helpers/PaginationHelper'
+} from '@venice/core/helpers/Pagination'
+import { IPagination } from '@venice/core/models'
+import styles from '@venice/styles/components/Pagination.module.scss'
+
 // TODO: these icons should be replaced when there is an icon library
 import AngleLeft from './icons/AngleLeft'
 import AngleRight from './icons/AngleRight'
 import PaginationIndex from './PaginationIndex'
 
 const debounce = (func: () => void, wait: number) => {
-  let timer: NodeJS.Timeout
+  let timer = 0
 
   return () => {
     clearTimeout(timer)
-    timer = setTimeout(func, wait)
+    timer = window.setTimeout(func, wait)
```

Comment:
We don't have any debounce function created?

Reply:
We have a [task ](https://juntossomosmais.monday.com/boards/346211397/pulses/637421651)to improve the Core folder and give support to code transpilation. Maybe the debounce function can be inserted when the task is completed.